ui/raidboss/data/07-dt/ultimate/futures_rewritten.ts

```diff
@@ -457,89 +460,109 @@ const triggerSet: TriggerSet<Data> = {
     },
     {
       id: 'FRU P1 Fall of Faith Collector',
-      type: 'StartsUsing',
+      type: 'Tether',
       netRegex: {
-        id: ['9CC9', '9CCC'],
+        id: ['00F9', '011F'], // 00F9 = fire; 011F = lightning
```

Comment:
Does this change have any impact on how soon this fires?

Reply:
Not substantively.  Technically, the `35` line shows up ~45ms earlier than the `20` line, but that won't be observably early or cause the alert to disappear early:
```
35|2025-01-15T17:42:54.1550000-08:00|400130CB|Fatebreaker's Image|10FF0001|Player1|0000|0000|00F9|
20|2025-01-15T17:42:54.2010000-08:00|400130CB|Fatebreaker's Image|9CC9|Fall of Faith|400130CB|Fatebreaker's Image|8.700|114.14|85.86|0.00|-0.79|
```